pkg/services/accesscontrol/models.go

```diff
@@ -341,16 +341,40 @@ const (
 
 	// Dashboard scopes
 	ScopeDashboardsAll = "dashboards:*"
+
+	// Alerting rules actions
+	ActionAlertingRuleCreate = "alert.rules:create"
+	ActionAlertingRuleRead   = "alert.rules:read"
+	ActionAlertingRuleUpdate = "alert.rules:update"
+	ActionAlertingRuleDelete = "alert.rules:delete"
+
+	// Alerting instances (+silences) actions
+	ActionAlertingInstanceCreate = "alert.instances:create"
+	ActionAlertingInstanceUpdate = "alert.instances:update"
+	ActionAlertingInstanceRead   = "alert.instances:read"
+
+	// Alerting Notification policies actions
+	ActionAlertingNotificationsCreate = "alert.notifications:create"
+	ActionAlertingNotificationsRead   = "alert.notifications:read"
+	ActionAlertingNotificationsUpdate = "alert.notifications:update"
+	ActionAlertingNotificationsDelete = "alert.notifications:delete"
+
+	// External alerting rule actions
+	ActionAlertingRuleExternalEdit = "alert.rules.external:edit"
```

Comment:
```suggestion
	ActionAlertingRuleExternalEdit = "alert.rules.external:write"
```

Maybe? I know it breaks consistency in favour of clarity but I'll leave that decision to you. The same comment applies to the the three others.

Reply:
Don't have an opinion on this. So, I am fine with the suggestion